pyxlsb/row.py

```diff
@@ -30,6 +30,6 @@ def __iter__(self):
     def _add_cell(self, col, *args, **kwargs):
         c = Cell(self, col, *args, **kwargs)
         self._cells[col] = c
-        if col > self._cols:
+        if col >= self._cols:
             self._cols = col
```

Comment:
There's 2 parts to this bug, the second is that we need `col + 1` here, `col` is an index, while `_cols` is a length.